### PR TITLE
feat: Add EdgeConv support for HeteroGraphConv

### PR DESCRIPTION
**Pull request by askorupka**
This PR adds EdgeConv support for HeteroGraphConv.
This is related to issue https://github.com/CarloLucibello/GraphNeuralNetworks.jl/issues/311.

### CODE REVIEW

File: test/layers/heteroconv.jl [hunk 1]
```diff
@@ -91,4 +91,13 @@
         output = y2.A[:, [2]]
         @test expected ≈ output
     end
+
+    @testset "CGConv" begin
```
Comment (CarloLucibello) [inline test/layers/heteroconv.jl:95]:
```suggestion
    @testset "EdgeConv" begin
```

**Comment by CarloLucibello** [comment]
needs a rebase

File: test/layers/heteroconv.jl (hunk 1 above)
Comment (askorupka) [reply test/layers/heteroconv.jl:95]: ah... it was late :joy_cat:

**Comment by askorupka** [comment]
@CarloLucibello rebased

**Pull request merged**